ts/src/bitmex.ts

```diff
@@ -2565,6 +2567,114 @@ export default class bitmex extends Exchange {
         };
     }
 
+    parseDepositWithdrawFee (fee, currency = undefined) {
+        //
+        //    {
+        //        asset: 'XBT',
+        //        currency: 'XBt',
+        //        majorCurrency: 'XBT',
+        //        name: 'Bitcoin',
+        //        currencyType: 'Crypto',
+        //        scale: '8',
+        //        enabled: true,
+        //        isMarginCurrency: true,
+        //        minDepositAmount: '10000',
+        //        minWithdrawalAmount: '1000',
+        //        maxWithdrawalAmount: '100000000000000',
+        //        networks: [
+        //            {
+        //                asset: 'btc',
+        //                tokenAddress: '',
+        //                depositEnabled: true,
+        //                withdrawalEnabled: true,
+        //                withdrawalFee: '20000',
+        //                minFee: '20000',
+        //                maxFee: '10000000'
+        //            }
+        //        ]
+        //    }
+        //
+        const networks = this.safeValue (fee, 'networks', []);
+        const networksLength = networks.length;
+        const result = {
+            'info': fee,
+            'withdraw': {
+                'fee': undefined,
+                'percentage': undefined,
+            },
+            'deposit': {
+                'fee': undefined,
+                'percentage': undefined,
+            },
+            'networks': {},
+        };
+        if (networksLength !== 0) {
+            const scale = this.safeString (fee, 'scale');
+            const precision = this.parsePrecision (scale);
+            for (let i = 0; i < networksLength; i++) {
+                const network = networks[i];
+                const networkId = this.safeString (network, 'asset');
+                const currencyCode = this.safeString (currency, 'code');
+                const networkCode = this.networkIdToCode (networkId, currencyCode);
+                const withdrawalFeeId = this.safeString (network, 'withdrawalFee');
+                const withdrawalFee = this.parseNumber (Precise.stringMul (withdrawalFeeId, precision));
```

Comment:
@AlgoQ we might need to add some networks inside options, we get non-unified ones 
```
                                 'minFee': '1000000',
                                 'tokenAddress': '8PPqvHmwwuWa4fc6YhZhueoBiaxX8r1yQnXX4MZGwjcN',
                                 'withdrawalEnabled': True,
                                 'withdrawalFee': '1000000'}],
                   'scale': '6'},
          'networks': {'ropsten': {'deposit': {'fee': None, 'percentage': None},
                                   'withdraw': {'fee': 25.0,
                                                'percentage': False}},
                       'tsol': {'deposit': {'fee': None, 'percentage': None},
                                'withdraw': {'fee': 1.0, 'percentage': False}},
                       'ttron': {'deposit': {'fee': None, 'percentage': None},
                                 'withdraw': {'fee': 1.0,
                                              'percentage': False}}},
          'withdraw': {'fee': None, 'percentage': None}}}
```